CHANGELOG.rst

```diff
@@ -11,6 +11,7 @@ Unreleased
 * Django 3.0, 3.1 and 3.2 support added
 * Python 3.5 and 3.6 support removed
 * Django 1.11 support removed
+* Hiding the back button in compare view
```

Comment:
This needs to be added to the unreleased section, the updating of the outdated branch resolved this.